pallets/identity/src/lib.rs

```diff
@@ -36,6 +36,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![recursion_limit = "256"]
 
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmarking;
```

Comment:
@vkomenda A general question about benchmarking: Should we move it into `test` folder?
I mean that functionality is not part of our library, It is an specialized test.
What do you think?

Reply:
Since benchmarking is only enabled when compiled with a feature flag, it's different from unit tests. We can follow the upstream code in adding `benchmarking.rs` to pallet crates. In the case of our `runtime` crate, a single module would have been overcrowded, which is why I added a special directory which is included only if the feature flag is set.